openshift/release/release-notes.md

```diff
@@ -0,0 +1,376 @@
+
+* :book: updating readme with 0.18 ([#3511](https://github.com/tektoncd/pipeline/pull/3511))
+
+<hr>
+
+# Tekton Pipelines v0.18.1
+
+## Fixes
+
+* :bug: [cherry-pick] Fix recursion issue on Skip ([#3534](https://github.com/tektoncd/pipeline/pull/3534))
+
+  Fix and issue in the pipeline state resolution which lead to very long or failed start times and high controller CPU in case of pipelines with a large number of dependencies between tasks (40+).
+
+* :bug: [cherry-pick] Change the webhook name to pipeline-webhook ([#3533](https://github.com/tektoncd/pipeline/pull/3533))
+
+  Fix an issue that caused the webhook, under certain conditions, to fail to acquire a lease and not function correctly as a result.
+<hr>
+
+# Tekton Pipelines v0.18.0
+
+## Features
+
+* :sparkles: Add readiness and liveness probes in controller ([#3489](https://github.com/tektoncd/pipeline/pull/3489))
+
+* :sparkles: Init conformance test ([#3400](https://github.com/tektoncd/pipeline/pull/3400))
+
+  Initialize the conformance test
+
+* :sparkles: Add timestaps in logs ([#3392](https://github.com/tektoncd/pipeline/pull/3392))
+  
+  Add timestamp in the logs of the tekton pipelines controller and webhook
+
+* :sparkles: Replicate webhook pods for resiliency ([#3391](https://github.com/tektoncd/pipeline/pull/3391))
+  
+  Add HorizontalPodAutoscaler, PodDisruptionBudget and podAntiAffinity to webhook
+
+* :sparkles: Add feature flag to disable creds-init ([#3379](https://github.com/tektoncd/pipeline/pull/3379))
+
+  Tekton's built-in credential mechanism can now be disabled by setting the disable-creds-init feature-flag to "true".
+
+* :sparkles: Fixes [#3086](https://github.com/tektoncd/pipeline/pull/3086) - amd64 images are pulled on all the architectures ([#3337](https://github.com/tektoncd/pipeline/pull/3337))
+  
+  The entrypoint lookup will pull the image for the controller's architecture instead of amd64 to better support different architectures
+
+* :sparkles: Update -shell-image to a multi-arch version ([#3334](https://github.com/tektoncd/pipeline/pull/3334))
+  
+  Use a multi-arch shell-image
+
+* :sparkles: When Expressions Status ([#3333](https://github.com/tektoncd/pipeline/pull/3333))
+
+  Resolved When Expressions are listed in the Skipped Tasks and the Task Runs sections of the Status
+
+* :sparkles: update submodules with --init flag ([#3321](https://github.com/tektoncd/pipeline/pull/3321))
+  
+  Git-init can now clone recursive submodules.
+
+* :sparkles: Upgrade knative -> 0.18 ([#3319](https://github.com/tektoncd/pipeline/pull/3319))
+  
+  This release will drop the resource_name label from webhook_request_(count, latencies_bucket) metrics. This this breaking change with no replacement for those relying on these metrics. The omission of these metrics will improve the overall memory usage of the webhook and the stability of the /metrics endpoint.
```

Comment:
Should this be documented in "breaking changes" ?

Reply:
There is no breaking changes section so added a word breaking change